src/main/java/com/microsoft/sqlserver/jdbc/SQLServerResource.java

```diff
@@ -652,5 +652,9 @@ static String getResource(String key) {
             {"R_unassignableError", "The class specified by the {0} property must be assignable to {1}."},
             {"R_InvalidCSVQuotes",
                     "Failed to parse the CSV file, verify that the fields are correctly enclosed in double quotes."},
-            {"R_TokenRequireUrl", "Token credentials require a URL using the HTTPS protocol scheme."},};
-};
+            {"R_TokenRequireUrl", "Token credentials require a URL using the HTTPS protocol scheme."},
+            {"R_maxResultBufferPropertyDescription",
```

Comment:
`R_maxResultBufferPropertyDescription` should describe what `maxResultBuffer` connection property does, please check  `R_useFmtOnlyPropertyDescription` for reference.  I'd suggest to create another String resource for `MaxResultBuffer property exceeded: {0}. MaxResultBuffer was set to: {1}.` and use `R_maxResultBufferPropertyDescription` for the actual description of the connection property.